app/services/analytics_events.rb

```diff
@@ -3488,9 +3488,9 @@ def cancel_account_reset_recovery
 
   # @param [Idv::ProofingComponentsLogging] proofing_components User's current proofing components
```

Comment:
Can we add a method comment annotation for the renamed event?

```suggestion
  # @identity.idp.previous_event_name IdV: Verify setup errors visited
  # @param [Idv::ProofingComponentsLogging] proofing_components User's current proofing components
```